randovania/games/samus_returns/json_data/header.json

```diff
@@ -0,0 +1,1711 @@
+                "long_name": "Spider Ball",
+                "max_capacity": 1,
+                "extra": {
+                    "item_id": "ITEM_SPIDER_BALL"
+                }
+            },
+            "Spring": {
+                "long_name": "Spring Ball",
+                "max_capacity": 1,
+                "extra": {
+                    "item_id": "ITEM_SPRING_BALL"
+                }
+            },
+            "MainPB": {
+                "long_name": "Main Power Bomb",
+                "max_capacity": 1,
+                "extra": {
+                    "item_id": "ITEM_WEAPON_POWER_BOMB"
+                }
+            },
+            "Boots": {
+                "long_name": "High Jump Boots",
+                "max_capacity": 1,
+                "extra": {
+                    "item_id": "ITEM_HIGH_JUMP_BOOTS"
+                }
+            },
+            "Space": {
+                "long_name": "Space Jump",
+                "max_capacity": 1,
+                "extra": {
+                    "item_id": "ITEM_SPACE_JUMP"
+                }
+            },
+            "Screw": {
+                "long_name": "Screw Attack",
+                "max_capacity": 1,
+                "extra": {
+                    "item_id": "ITEM_SCREW_ATTACK"
+                }
+            },
+            "Baby": {
+                "long_name": "Baby Metroid",
+                "max_capacity": 1,
+                "extra": {
+                    "item_id": "ITEM_BABY_HATCHLING"
+                }
+            },
+            "ETank": {
+                "long_name": "Energy Tank",
+                "max_capacity": 10,
+                "extra": {
+                    "item_id": "ITEM_ENERGY_TANKS"
+                }
+            },
+            "MissileAmmo": {
+                "long_name": "Missile",
+                "max_capacity": 999,
+                "extra": {
+                    "item_id": "ITEM_WEAPON_MISSILE_CURRENT",
+                    "item_capacity_id": "ITEM_WEAPON_MISSILE_MAX"
+                }
+            },
+            "SMAmmo": {
+                "long_name": "Super Missile",
+                "max_capacity": 99,
+                "extra": {
+                    "item_id": "ITEM_WEAPON_SUPER_MISSILE_CURRENT",
+                    "item_capacity_id": "ITEM_WEAPON_SUPER_MISSILE_MAX"
+                }
+            },
+            "PBAmmo": {
+                "long_name": "Power Bomb",
+                "max_capacity": 99,
+                "extra": {
+                    "item_id": "ITEM_WEAPON_POWER_BOMB_CURRENT",
+                    "item_capacity_id": "ITEM_WEAPON_POWER_BOMB_MAX"
+                }
+            },
+            "Aeion": {
+                "long_name": "Aeion",
+                "max_capacity": 2000,
+                "extra": {
+                    "item_id": "ITEM_CURRENT_SPECIAL_ENERGY",
+                    "item_capacity_id": "ITEM_MAX_SPECIAL_ENERGY"
+                }
+            },
+            "Command Visor": {
```

Comment:
?

Reply:
PlayerShipNode bullshit that's getting fixed, but hasn't yet

Reply:
Temporary fix until PlayerShipNode was updated. Games wouldn't generate without it.